### PR TITLE
chore: Update package imports to absolute

### PR DESCRIPTION
**Pull request by deepak1725**
Relative Imports are not understood by IDE's and the correct importing of custom Modules was kind of shooting in a dark.

#### Before:
before

#### After:
after

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM registry.centos.org/centos/centos:7
```
Comment (arajkumar) [inline Dockerfile:2]: Same changes has to be repeated for Dockerfile.rhel and Dockerfile.new

File: Dockerfile [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
   
 ENV APP_DIR='/f8a_report'
-
+ENV PYTHONPATH=.
 WORKDIR ${APP_DIR}
```
Comment (arajkumar) [inline Dockerfile:5]:
```suggestion
ENV PYTHONPATH=${APP_DIR}
```

**Review by arajkumar — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 2 above)
Comment (deepak1725) [reply Dockerfile:5]: Can't replace WORKDIR with said change

The next steps are failing:
RUN pip3 install -r requirements.txt
CMD ["f8a_report/stack_report_main.py"]

File: Dockerfile (hunk 1 above)
Comment (deepak1725) [reply Dockerfile:2]: Ack

**Review by arajkumar — APPROVED** [review]
lgtm

**Pull request merged**